bin/genn-buildmodel.sh

```diff
@@ -52,13 +52,20 @@ done
 if [[ -z "$MODEL" ]]; then
     genn_error $LINENO 2 "no model file given"
 fi
-pushd $OUT_PATH > /dev/null
-OUT_PATH="$PWD"
+
+# Use pushd to get an absolute path and symbolic links in /tmp to avoid problems
+# with spaces in the path names 
+pushd "$OUT_PATH" > /dev/null
+OUT_PATH=/tmp/genn_outpath$RANDOM
```

Comment:
It would be _marginally_ nicer to use ``OUTPUT=$(mktemp)`` (and ``MODEL=$(mktemp)`` instead). Better chance of being robust to multiple users running GeNN simultaneously on the same machine.